packages/perseus/src/components/input-with-examples.tsx

```diff
@@ -146,9 +146,18 @@ class InputWithExamples extends React.Component<Props, State> {
     render(): React.ReactNode {
         const input = this._renderInput();
 
-        const examplesContent = _.map(this.props.examples, (example) => {
-            return "- " + example;
-        }).join("\n");
+        const examplesContent =
+            this.props.examples.length <= 2
+                ? this.props.examples.join(" ") // A single item (with or without leading text) is not a "list"
+                : this.props.examples // 2 or more items should display as a list
+                      .map((example, index) => {
+                          // If the first example is bold, then it is most likely a heading/leading text.
+                          // So, it shouldn't be part of the list.
+                          return index === 0 && example.startsWith("**")
```

Comment:
It looks like both remaining usages of `<InputWithExamples />` provides an examples list where the first item is indeed a bold heading ([here](https://github.com/Khan/perseus/blob/9c2289b3fb0e415c4cef979ab146dc583ef91b7d/packages/perseus/src/widgets/input-number/input-number.tsx#L181) and [here](https://github.com/Khan/perseus/blob/94eba15cacf904daabce378d080291613490c4a1/packages/perseus/src/widgets/numeric-input/numeric-input.tsx#L132)). 

Perhaps we can simplify this once `input-number` is truly gone because at that point we'll know exactly what is always passed in.

Reply:
That's the plan. We have follow-up tickets to refactor aspects of Numeric Input. Seeing as how `InputWithExamples` isn't used anywhere else, we are considering bringing this component into Numeric Input and simplifying the two code bases.